bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala

```diff
@@ -65,8 +65,8 @@ case class Semester(year: Int, half: SemesterOption) {
     cal.setTimeZone(TimeZone.getTimeZone("UTC"))
     cal.set(Calendar.YEAR, year)
     cal.set(Calendar.MONTH, half.firstMonth)
-    cal.set(Calendar.DAY_OF_MONTH, 0)
-    cal.set(Calendar.HOUR, 0)
+    cal.set(Calendar.DAY_OF_MONTH, 1)
+    cal.set(Calendar.HOUR_OF_DAY, 0)
```

Comment:
Interesting. I didn't know this. Should be easy enough to fix, of course, and it makes more sense than starting at midnight given observing would never start that late under acceptable conditions.

As Carlos said, how does this figure into the end of semester? I am assuming the same, too, then, regarding 2:00 PM on July 31?